### PR TITLE
Swift 3.2 compatibility.

### PR DESCRIPTION
**Pull request by AtomicCat**
- Cast value to UInt, as Xcode9b5 compiler appears to default value to an Int and then errors on mixing UInt and Int operations.
- Accept all Xcode9 suggested compiler warning settings.
- Set SWIFT_VERSION to 3.2 in all targets. All targets still build with Swift 3.1/Xcode 8.3.x.
- Rename Network example framework to WebServices to avoid a name collision with an iOS 11 private framework with the same name.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [3.0.1](https://github.com/Nike-Inc/Willow/releases/tag/3.0.1)
+
+#### Updated
+
+- Fix compile error for Swift 3.2/4.0.
+- Rename Network example framework to WebServices to avoid a name collision with an iOS 11 private framework with the same name.
+- Set SWIFT_VERSION to 3.2 in all targets. All targets still build with Swift 3.1/Xcode 8.3.x.
```
Comment (cnoon) [inline CHANGELOG.md]: These should all probably be written in past tense. I'd also probably change the `Fix compile error` to something more like: `Updated source and test code to compile against Swift 3.2 and 4.0.`.

File: Example/Frameworks/WebServices/WebServices.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //
-//  Network.h
+//  WebServices.h
 //
 //  Copyright (c) 2015-2016 Nike, Inc. (https://www.nike.com)
```
Comment (cnoon) [inline Example/Frameworks/WebServices/WebServices.h]: Could you bump `2016` to `2017` where applicable?

File: Example/iOS Example/ViewController.swift [hunk 3]
```diff
@@ -23,7 +23,7 @@
 //
 
 import Database
-import Network
+import WebServices
 import UIKit
 import Willow
```
Comment (cnoon) [inline Example/iOS Example/ViewController.swift]: Alphabetical?

File: Example/iOS Example/WillowConfiguration.swift [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
 import Database
 import Foundation
-import Network
+import WebServices
 import UIKit
 import Willow
```
Comment (cnoon) [inline Example/iOS Example/WillowConfiguration.swift]: Alphabetical?

File: Example/iOS Example/WillowConfiguration.swift [hunk 5]
```diff
@@ -65,7 +65,7 @@ struct WillowConfiguration {
     static func configure(
         appLogLevels: LogLevel = [.debug, .info, .event],
         databaseLogLevels: LogLevel = [.sql, .debug, .info, .event],
-        networkLogLevels: LogLevel = [.debug, .info, .event],
+        webservicesLogLevels: LogLevel = [.debug, .info, .event],
```
Comment (cnoon) [inline Example/iOS Example/WillowConfiguration.swift]: `webservices` -> `webServices`?

File: Source/LoggerConfiguration.swift [hunk 6]
```diff
@@ -75,7 +75,7 @@ public struct LoggerConfiguration {
             var specifiedValues: [LogLevel: [T]] = [:]
 
             for bitShift in UInt(0)..<UInt(32) {
-                let bitValue = 1 << bitShift
+                let bitValue = UInt(1) << bitShift
```
Comment (cnoon) [inline Source/LoggerConfiguration.swift:78]: 👍🏻

File: Willow.podspec [hunk 7]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'Willow'
-  s.version = '3.0.0'
+  s.version = '3.0.1'
```
Comment (cnoon) [inline Willow.podspec:3]: I think you might have missed the `Info.plist` and `Info-tvOS.plist` files...

**Review by cnoon — CHANGES_REQUESTED** [review]
Overall it looks great @AtomicCat! Just a few minor comments and we can get it pushed out.

File: Willow.podspec (hunk 7 above)
Comment (AtomicCat) [reply Willow.podspec:3]: fixed

File: Source/LoggerConfiguration.swift (hunk 6 above)
Comment (AtomicCat) [inline Source/LoggerConfiguration.swift]: 👍

**Review by cnoon — APPROVED** [review]
Looks good @AtomicCat! 🍻

**Pull request merged**